packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -81,19 +87,19 @@ The `useErrorBoundaryFallbackProps` allows you to access the `reset` method and
 ```tsx
 import { ErrorBoundary, useErrorBoundaryFallbackProps } from '@suspensive/react'
 
-const Fallback = () => {
-  const props = useErrorBoundaryFallbackProps()
+const Nested = () => {
```

Comment:
I think this component should be named as Nested, not just Fallback because it's contained in ErrorBoundaryFallback